MaxText/train.py

```diff
@@ -289,6 +289,9 @@ def loss_fn(model, config, data, dropout_rng, params, is_train=True):
   if is_train:
     for k, v in data.items():
       data[k] = v[: config.micro_batch_size_to_train_on, :]
+  else:
```

Comment:
This `is_train` is set to False when running decoding. Are we sure we want to clip the data when running inference ?